sdk/core/src/main/java/org/apache/mesos/dcos/DCOSCertInstaller.java

```diff
@@ -0,0 +1,83 @@
+package org.apache.mesos.dcos;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Facilitates installation of DC/OS cert inside framework's JRE.
+ */
+public class DCOSCertInstaller {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DCOSCertInstaller.class);
+
+    public static final String DEFAULT_JRE_KEYSTORE_PASS = "changeit";
+    public static final String CERT_PATH = ".ssl/ca.crt";
+
+    public static boolean installCertificate(String pathToJRE) {
+        return installCertificate(pathToJRE, new ProcessRunner());
+    }
+
+    @VisibleForTesting
+    protected static boolean installCertificate(String pathToJRE, ProcessRunner processRunner) {
+        try {
+            final Path jrePath = Paths.get(pathToJRE);
+            // Check if JRE is present, there's nothing to do.
+            if (!Files.exists(jrePath)) {
+                LOGGER.info("JRE not found at path: {}", jrePath.toAbsolutePath());
+                return false;
+            }
+
+            final Path sandboxCertPath = Paths.get(CERT_PATH);
+            // Check if cert is present, there's nothing to do.
+            if (!Files.exists(sandboxCertPath)) {
+                LOGGER.info("Cert file not found at path: {}", sandboxCertPath.toAbsolutePath());
+                return false;
+            }
+
+            final Path jreDefaultKeystorePath = Paths.get(pathToJRE, "/lib/security/cacerts");
```

Comment:
`keytool` resides in the same `/bin` directory as `java` executable, and shares the same permissions. And, because the scheduler is run using `java` executable, the fact that we reach this line of code, means that we'll not have trouble running it.